requirements/audio.txt

```diff
@@ -6,3 +6,6 @@ pesq >=0.0.4, <0.0.5
 pystoi >=0.3.0, <0.5.0
 torchaudio >=0.10.0, <2.5.0
 gammatone >=1.0.0, <1.1.0
+librosa >=0.9.0, <0.11.0
+onnxruntime-gpu >=1.12.0, <1.19; sys_platform != 'darwin'
```

Comment:
@quancs does this need to be GPU's one?

Reply:
The CPU one is OK, but is slower. Anyway, let's use the cpu one for test